server/websocket.go

```diff
@@ -1367,7 +1367,18 @@ func (c *client) wsCollapsePtoNB() (net.Buffers, int64) {
 		}
 		var csz int
 		for _, b := range nb {
-			cp.Write(b)
+			for len(b) > 0 {
+				n, err := cp.Write(b)
+				if err != nil {
```

Comment:
Can this be a short write error as well?

Reply:
Don't believe so, no, doesn't look like the compressor library ever returns that error code.